Readme.md

```diff
@@ -2,31 +2,106 @@
+[https://github.com/aquasecurity/tracee](https://github.com/aquasecurity/tracee).
 
-libbpfgo is built around libbpf - the standard library for interacting with eBPF from userspace, which is a C library maintained in Linux upstream. We have created libbpfgo as a thin Go wrapper around libbpf.
+libbpfgo is built around [libbpf](https://github.com/libbpf/libbpf) - the
+standard library for interacting with eBPF programs from userspace - which is a
+C library maintained in Linux upstream. We have created libbpfgo as a thin Go
+wrapper around the libbpf project.
 
 ## Installing
 
-libbpfgo is using CGO to interop with libbpf and will expect to be linked with libbpf at run or link time. Simply importing libbpfgo is not enough to get started, and you will need to fulfill the required dependency in one of the following ways:
+libbpfgo uses CGO to interop with libbpf and will expect to be linked with
+libbpf at run or link time. Simply importing libbpfgo is not enough to get
+started, and you will need to fulfill the required dependency in one of the
+following ways:
 
-1. Install the libbpf as a shared object in the system. Libbpf may already be packaged for you distribution, if not, you can build and install from source. More info [here](https://github.com/libbpf/libbpf).
-1. Embed libbpf into your Go project as a vendored dependency. This means that the libbpf code is statically linked into the resulting binary, and there are no runtime dependencies. [Tracee](https://github.com/aquasecurity/tracee) takes this approach and you can take example from it's [Makefile](https://github.com/aquasecurity/tracee/blob/f8df7da6a27f729610992b6bd52e89d510fcf384/tracee-ebpf/Makefile#L62).
+1. Install libbpf as a shared object in the system. Libbpf may already be
+   packaged for your distribution and, if not, you can build and install from
+   source. More info [here](https://github.com/libbpf/libbpf).
+1. Embed libbpf into your Go project as a vendored dependency. This means that
+   the libbpf code is statically linked into the resulting binary, and there
+   are no runtime dependencies.
+   [Tracee](https://github.com/aquasecurity/tracee) takes this approach.
+
+In the next sesssion you will find different ways to build libbpfgo.
+
+## Building
+
+Currently you will find the following GNU Makefile rules:
+
+| Makefile Rule &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Description &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; |
```

Comment:
Do we need all these non breaking spaces? I think github table rendering will render it the same way without them. See [link](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#creating-a-table).

Reply:
The 3 tables got pretty unaligned when rendering.. I tried different ways to have the 3 tables aligned and that was the only way. What do you think ? Should we keep them unaligned ? MDs are usually read through github interface was my thought.

Reply:
by unaligned you mean they weren't the same width? if so, that's fine (the table adjusts the the width of the line). we wouldn't want to now manually size the tables

Reply:
Alright, I fixed the ugly spaces. Marking as solved as I'll push the changes at the end.